schedule/src/Views/EventDetail.elm

```diff
@@ -155,6 +167,13 @@ eventMetaDataSidebar event =
                             False ->
                                 []
                        )
+                    ++ (case event.slidesUrl of
+                            Nothing ->
+                                []
+
+                            Just url ->
+                                [ li [] [ strong [] [ text "Recording: " ], a [ href url ] [ text event.slidesFilename ] ] ]
+                       )
```

Comment:
This block doesn't seem to fit in. We already show a download link if there is any slides in line 136. Or am I missing something?